src/components/constants.ts

```diff
@@ -1,8 +1,17 @@
-export const KeyCode = {
+export enum EKeyCode {
```

Comment:
Why introduce this enum?

Reply:
Because I needed to use KeyCode keys as type. Something like `keyof typeof KeyCode` seemed complex

Reply:
Reverted